stdlib/public/SwiftShims/RefCount.h

```diff
@@ -432,7 +431,14 @@ class RefCountBitsT {
   }
 
   LLVM_ATTRIBUTE_ALWAYS_INLINE
-  RefCountBitsT(RefCountBitsT<RefCountIsInline> newbits) {
+  RefCountBitsT(int dummy, RefCountBitsT<RefCountIsInline> newbits) {
```

Comment:
This is actually a Clang bug - GCC and MSVC correctly reject this code! We should definitely fix this, but in a nicer way!

http://bugs.llvm.org/show_bug.cgi?id=32074